implementations/rust/ockam/ockam_node/src/storage/database/migrations/node_migrations/sql/postgres/20250115100000_create_database.sql

```diff
@@ -67,7 +71,8 @@ CREATE TABLE credential
     scope              TEXT  NOT NULL,
     credential         BYTEA NOT NULL,
     expires_at         INTEGER,
-    node_name          TEXT  NOT NULL  -- node name to isolate credential that each node has
+    node_name          TEXT  NOT NULL,  -- node name to isolate credential that each node has
+    PRIMARY KEY (tenant_id, subject_identifier, issuer_identifier, scope)
 );
 
 CREATE UNIQUE INDEX credential_issuer_subject_scope_index ON credential (issuer_identifier, subject_identifier, scope);
```

Comment:
I _think_ this is ok,   these IIUC are credentials a node has retrieved for itself (not credentials presented by other nodes).  So,  `(issuer_identifier, subject_identifier)`   looks to be unique,  at least if there aren't two nodes with same identity.    But.. could we include the tenant_id in the unique index so it is the same than all others?.   I'm not sure if we couldn't end up in a situation where there are different nodes with same identity (I mean.. again that _probably_ is fine here,  but hard to reason about)

Reply:
Looking at the full source there are two index:

```
CREATE UNIQUE INDEX credential_issuer_subject_scope_index ON credential (issuer_identifier, subject_identifier, scope);
CREATE UNIQUE INDEX credential_issuer_subject_index ON credential (issuer_identifier, subject_identifier);
```
Not sure what's the semantic of scope there,  but only 1 of these unique index make sense to keep

Reply:
Fixed both cases.